Library/Homebrew/cmd/doctor.rb

```diff
@@ -424,10 +424,18 @@ def check_access_share_man
     __check_subdir_access "share/man"
   end
 
+  def check_access_homebrew_repository
+    unless HOMEBREW_REPOSITORY.writable_real? then <<-EOS.undent
+      The #{HOMEBREW_REPOSITORY} is not writable.
+      You should probably `chown` #{HOMEBREW_REPOSITORY}
```

Comment:
"You should probably `chown`" is used multiple times in `doctor`. I think we should be consistent. 

Also, there are much point to mention sudo, unless we show complete command.